pkg/remoteconfig/state/products/apmsampling/remote_rates.go

```diff
@@ -27,7 +27,21 @@ type TargetTPS struct {
 	Mechanism SamplingMechanism `msgpack:"4"`
 }
 
-// APMSampling is the list of target tps
+type RareSamplerConfig string
+
+const (
+	RareSamplerConfigUnset    RareSamplerConfig = "" // default
+	RareSamplerConfigEnabled  RareSamplerConfig = "ENABLED"
+	RareSamplerConfigDisabled RareSamplerConfig = "DISABLED"
+)
+
+type ErrorSamplerConfig struct {
+	TargetTPS float64 `msgpack:"targetTPS"`
+}
+
+// APMSampling contains values for remotely configuring sampling
 type APMSampling struct {
-	TargetTPS []TargetTPS `msgpack:"0"`
+	TargetTPS           []TargetTPS         `msgpack:"0"`
+	RareSamplerConfig   RareSamplerConfig   `msgpack:"rareSamplingConfig"`
```

Comment:
Can we use a struct instead of a single string here?. In the future, if we want to add a new config to `RareSamplerConfig`, it will be easier.